Realm/RLMSyncUtil.mm

```diff
@@ -19,6 +19,29 @@
 #import <Foundation/Foundation.h>
 
 #import "RLMSyncUtil_Private.hpp"
+#import "RLMSyncUser_Private.hpp"
+#import "RLMRealmConfiguration+Sync.h"
+#import "RLMRealmConfiguration_Private.hpp"
+#import "RLMSyncPermissionChange.h"
+
+@implementation RLMRealmConfiguration (RealmSync)
++ (instancetype)managementConfigurationForUser:(RLMSyncUser *)user {
+    NSURLComponents *components = [NSURLComponents componentsWithURL:user.authenticationServer resolvingAgainstBaseURL:NO];
+    if ([components.scheme isEqualToString:@"https"]) {
+        components.scheme = @"realms";
+    } else {
+        components.scheme = @"realm";
+    }
+    components.path = @"/~/__management";
+    NSURL *managementRealmURL = components.URL;
+    RLMSyncConfiguration *syncConfig = [[RLMSyncConfiguration alloc] initWithUser:user realmURL:managementRealmURL];
+    RLMRealmConfiguration *config = [RLMRealmConfiguration new];
+    config.syncConfiguration = syncConfig;
+    config.objectClasses = @[RLMSyncPermissionChange.class];
+    config.schemaMode = realm::SchemaMode::Additive;
```

Comment:
This is set automatically when a sync config is set (the private setter is just there for testing it without using sync).